### PR TITLE
Update find-and-release-quarantined-messages-as-a-user.md

### PR DESCRIPTION
**Pull request by faoquong**
@chrisda

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/find-and-release-quarantined-messages-as-a-user.md [hunk 1]
```diff
@@ -152,6 +152,8 @@ After you select a message, you have options for what to do with the messages in
 
 - **Remove from quarantine**: After you click **Yes** in the warning that appears, the message is immediately deleted.
 
+- **Block Sender**: this blocks the sender from sending emails to user.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/find-and-release-quarantined-messages-as-a-user.md]:
```suggestion
- **Block Sender**: This blocks the sender from sending emails to you.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@faoquong Please update with the feedback. Thanks.

**Pull request merged**